__tests__/project.test.js

```diff
@@ -34,3 +34,29 @@ beforeEach(() => {
         });
     })
   });
+
+  describe("GET request along the path of /api/article/:article_id.. returns an object of an article", () => {
+    test(" get an article object with the status code 200", () => {
+      const article_Id=1;
+      return request(app)
+        .get(`/api/article/${article_Id}`)
+        .expect(200)
+        .then((response) => {
+          expect(response.body.article).toHaveProperty("author", "butter_bridge");
+          expect(response.body.article).toHaveProperty("title", "Living in the shadow of a great man");
+          expect(response.body.article).toHaveProperty("body", "I find this existence challenging");
+          expect(response.body.article).toHaveProperty("topic", "mitch");
+          expect(response.body.article).toHaveProperty("created_at", expect.any(String));
+          expect(response.body.article).toHaveProperty("votes", expect.any(Number));
+        });
+    });
+    test(" try to get an article object that does not exist giving a status 200", () => {
+      const article_Id=99999;
```

Comment:
For the test beginning on line 53 - if an article does not exist (we get nothing back from our query) then we would want to expect to get a status 404 not found error message, not a 200 here